knotx-core/knotx-api/src/main/java/com/cognifide/knotx/api/EmptyRepositoryResponse.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Knot.x - Reactive microservice assembler - API
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.api;
+
+import io.vertx.rxjava.core.MultiMap;
+
+public class EmptyRepositoryResponse extends RepositoryResponse {
```

Comment:
We should provide 2 additional subclasses of RepositoryResponse:
DynamicRepositoryResponse (Repository knows that requested response contains dynamic snippets)
StaticRepositoryResponse (Repository knows that requested response does not contain dynamic snippets)